mobileApp/jobs/buildWhiteLabelAndroidApps.groovy

```diff
@@ -8,10 +8,16 @@ Example secret YAML file used by this script
 jobConfig:
     jobName: build-demo-android-app
     public: true
+    release: false
```

Comment:
@benpatterson We usually add the comments to the actual secret files. Do you think we should put them here too?